src/chart/generateCategoricalChart.tsx

```diff
@@ -1138,7 +1129,7 @@ export const generateCategoricalChart = ({
           top: this.props.margin.top ?? 0,
         },
         coordinateList: this.state.tooltipTicks,
-        mouseHandlerCallback: this.handleMouseMove,
+        mouseHandlerCallback: this.accessibilityMouseHandler,
```

Comment:
I don't think this is accurate - mouseHandlerCallback should still be handleMouseMove

Reply:
Since triggeredAfterMouseMove is throttled, it doesn't seem appropriate here.